src/Core/Models/TimeOfDay.php

```diff
@@ -11,16 +11,62 @@
 use Exception;
 use JsonSerializable;
 
-class TimeOfDay extends DateTime implements JsonSerializable
+/**
+ * This object represents time in hours minutes and seconds
+ */
```

Comment:
```suggestion
/**
 * Class TimeOfDay
 * 
 * This object represents time in hours minutes and seconds
 * 
 * @package Microsoft\Graph\Core\Models
 * @copyright 2021 Microsoft Corporation
 * @license https://opensource.org/licenses/MIT MIT License
 * @link https://developer.microsoft.com/graph
 */
```
I use this doc template for classes/interfaces in PHPStorm (Settings > Editor > General > File and Code Templates):
```
/**
 * Class ${NAME}
#if (${NAMESPACE}) * @package ${NAMESPACE}
#end
 * @copyright ${YEAR} Microsoft Corporation
 * @license https://opensource.org/licenses/MIT MIT License
 * @link https://developer.microsoft.com/graph
 */
```

Reply:
Any reasons for having 2 copyright notices? I have looked at the other SDKs and the format doesn't look like this.